src/main/resources/application.conf

```diff
@@ -81,6 +81,8 @@ metrics {
     port = ${?DATADOG_AGENT_PORT}
     tracked_consumer_group = []
     tracked_consumer_group =  ${?DATADOG_CONSUMER_GROUPS}
+    removeTagsFromMetricName = false
```

Comment:
A nitpick, but it would probably make sense to make the config param naming consistent, so a snake case rather than camel case: `remove_tags_from_metric_name`

Reply:
👍